files/store.go

```
@@ -90,6 +90,13 @@ func (s Store) UploadFile(ctx context.Context, metadata files.FileMetaData, resu
 
 	if response.AllPartsUploaded {
 		var err error
+		if metadata.Path != "" {
+			existingFile, _ := s.files.GetFile(ctx, metadata.Path, s.cfg.ServiceAuthToken)
+			if existingFile.Path == metadata.Path {
+				log.Error(ctx, "duplicate file detected in upload service", ErrFilesAPIDuplicateFile, log.Data{"path": metadata.Path})
+				return false, ErrFilesAPIDuplicateFile
+			}
+		}
```

The error is being ignored from `GetFIle`, I think it should be checked.
Should that log event message say `files-api` instead of `upload service`?